schema.sql

```diff
@@ -1,5 +1,11 @@
-/* Database schema to keep the structure of entire database. */
+CREATE TABLE animals ( ID int, Name varchar(100), Date_of_birth date, Escape_attempts int, Neutered boolean, Weight_kg decimal );
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+ALTER TABLE table_name ADD PRIMARY KEY (column_name);
+
+ALTER TABLE animals DROP COLUMN ID; -----> deletes a specific column(ID)
+
+ALTER TABLE animals ADD ID SERIAL PRIMARY KEY;
+
+ALTER DATABASE vet_clinic DROP TABLE animals; ------> deletes the database's table animals
+
+ALTER TABLE animals 
```

Comment:
@silviatofana  great work implementing the changes requested, as previously request kindly remove the queries.